scripts/kiln.lua

```diff
@@ -230,11 +235,15 @@ end
 function closePopUp()
     while 1 do
         srReadScreen()
-        local ok = srFindImage("OK.png")
+		local outofresource = srFindImage("YouDont.png");
+        local ok = srFindImage("OK.png");

```

Comment:
Looks like some space vs tab indentation going slightly wrong or something making this hard to read, can you do a quick fix for that (here, above, and below in this file)?  I think we're using spaces for indentation in this repo, but I'm not 100% sure ^_^.

Reply:
Sorry about that, I've done a new commit converting all my tabs to spaces.

Reply:
No worries, and, thanks!